lib/src/external_module.cc

```diff
@@ -313,8 +315,10 @@ ActionResponse ExternalModule::callBlockingAction(const ActionRequest& request)
     LOG_TRACE("Input for the %1%: %2%", request.prettyLabel(), action_args);
 
     auto exec = lth_exec::execute(
-#ifdef _WIN32
+#if defined(_WIN32)
         "cmd.exe", { "/c", path_, action_name },
+#elif defined(__sun)
+        "/usr/bin/ctrun", { "-l", "child", path_, action_name },
```

Comment:
I'm not sure we've been consistent about that. Also pinging @peterhuene.

I think we've tended to use absolute paths for things that are intrinsic to the OS, and relative paths for packages that might be installed as dependencies (and we've been moving to allowing an absolute path to be specified at compile-time for those as we move them into the puppet-agent package).